torchtune/modules/tokenizer.py

```diff
@@ -40,6 +44,9 @@ def __init__(
         self.bos_id = bos_id
         self.eos_id = eos_id
         self.pad_id = pad_id
+        self.encodes_whitespace = any(
+            [self.spm_model.encode(c) for c in WHITESPACE_CHARS]
+        )
```

Comment:
no idea what's going on here...

Reply:
I should probably add a comment

Reply:
But yeah, discussed the purpose of this variable in the second point [here](https://github.com/pytorch/torchtune/pull/624/files#r1547101264)